example-get-started/code/.github/workflows/cml.yaml

```
@@ -11,21 +11,21 @@ jobs:
           REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           cml ci
-          echo "# CML Report" > report.md
+          if [ "$GITHUB_REF" = "refs/heads/main" ] ; then
+            PREVIOUS_REF='HEAD~1'
+          else
+            PREVIOUS_REF='main'
+          fi
```

```suggestion
          if [ "$GITHUB_REF" = refs/heads/main ]; then
            PREVIOUS_REF=HEAD~1
          else
            PREVIOUS_REF=main
          fi
```

Optional nitpick: remove unnecessary quotes and space before semicolon.